cmd/template/main.go

```diff
@@ -48,3 +48,33 @@ clean:
 .PHONY: all build run test clean
 		`)
 }
+
+func ReadmeTemplate() []byte {
+	return []byte(
+		`
+# Project Title
+
+One Paragraph of project description goes here
+
```

Comment:
An idea here would be to template the ProjectName from the struct into this so instead of Project Title we can use the project name here
```{{.Pojrectname}}```

Reply:
see latest commit